ecosystem/typescript/sdk/src/aptos_account.ts

```diff
@@ -48,21 +73,47 @@ export class AptosAccount {
     return this.authKeyCached;
   }
 
-  /** Returns the public key for the associated account */
+  /**
+   * This key is generated with Ed25519 scheme.
+   * Public key is used to check a signature of transaction, signed by given account
+   * @returns The public key for the associated account
+   */
   pubKey(): HexString {
     return HexString.ensure(Buffer.from(this.signingKey.publicKey).toString('hex'));
   }
 
+  /**
+   * Signs specified `buffer` with account's private key
+   * @param buffer A buffer to sign
+   * @returns A signature HexString
+   */
   signBuffer(buffer: Buffer): HexString {
     const signature = Nacl.sign(buffer, this.signingKey.secretKey);
     return HexString.ensure(Buffer.from(signature).toString('hex').slice(0, 128));
   }
 
+  /**
+   * Signs specified `hexString` with account's private key
+   * @param hexString A regular string or HexString to sign
+   * @returns A signature HexString
+   */
   signHexString(hexString: MaybeHexString): HexString {
     const toSign = HexString.ensure(hexString).toBuffer();
     return this.signBuffer(toSign);
   }
 
+  /**
+   * Derives account address, public key and private key
+   * @returns AptosAccountObject instance.
+   * @example An example of the returned AptosAccountObject object
+   * ```
+   * {
+   *    address: "0xe8012714cd17606cee7188a2a365eef3fe760be598750678c8c5954eb548a591",
+   *    publicKeyHex: "0xf56d8524faf79fbc0f48c13aeed3b0ce5dd376b4db93b8130a107c0a5e04ba04 ",
+   *    privateKeyHex: "0x009c9f7c992a06cfafe916f125d8adb7a395fca243e264a8e56a4b3e6accf940d2b11e9ece3049ce60e3c7b4a1c58aebfa9298e29a30a58a67f1998646135204"
```

Comment:
// ts-ignore?

Reply:
Seems like eslint does not recognize `eslint-disable` within TSDoc comments. I decided to just use template string literal